Assets/VRTK/Scripts/Controls/2D/PanelMenu/PanelMenuItemController.cs

```diff
@@ -0,0 +1,174 @@
+// Panel Menu Item Controller
```

Comment:
same on this one as well, use this as an example: 

https://github.com/thestonefox/SteamVR_Unity_Toolkit/blob/master/Assets/VRTK/Scripts/Controls/2D/RadialMenu/RadialMenu.cs